cicd/forgeops-tests/configure.py

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env python3
+import sys
+import os
+import shutil
+import yaml
+import re
+
+TESTS_NAMESPACE = "smoke"
+if "TESTS_NAMESPACE" in os.environ:
+    TESTS_NAMESPACE = os.environ["TESTS_NAMESPACE"]
+
+TESTS_DOMAIN = "forgeops.com"
+if "TESTS_DOMAIN" in os.environ:
+    TESTS_DOMAIN = os.environ["TESTS_DOMAIN"]
+
+TESTS_IMAGE_TAG = "7.0.0-SNAPSHOT"
+if "TESTS_IMAGE_TAG" in os.environ:
+    TESTS_IMAGE_TAG = os.environ["TESTS_IMAGE_TAG"]
+
+TESTS_DEPLOYMENT = "smoke-deployment"
+if "TESTS_DEPLOYMENT" in os.environ:
+    TESTS_DEPLOYMENT = os.environ["TESTS_DEPLOYMENT"]
+
+TESTS_COMPONENTS = "openam amster"
+if "TESTS_COMPONENTS" in os.environ:
+    TESTS_COMPONENTS = os.environ["TESTS_COMPONENTS"]
+
```

Comment:
I cannot see where the following variables are used.

Reply:
Line 80 => `    for component in TESTS_COMPONENTS.split(" "):`